docs/core-concepts/core-concepts-overview.md

```diff
@@ -0,0 +1,199 @@
+---
+id: core-concepts
+title: Core Concepts
+sidebar_label: Core Concepts
+---
+
+COMIT is an open protocol facilitating cross-blockchain applications.
+This part of the documentation explains core concepts and how they relate to the COMIT protocol.
+The documentation contains embedded google slides for explaining details.
+Note that you can play/pause the slide-player and step back and forth. 
+
+More detailed information about the COMIT protocol can be found in [COMIT protocol stack](comit-protocol-stack.md).
+
+## Atomic Swaps using HTLCs
+
+[Atomic Swap](#atomic-swap)s using [HTLC](#htlc)s defines solution using time-locks for exchanging two digital assets without the need of a trusted third party.
+
+Prior to the actual swap execution there is usually a [Negotiation](#negotiation---maker-and-taker) phase where the two parties agree on the ledgers, assets and amounts to be swapped.
+
+Assuming, that Alice has ETH on Ethereum and Bob has BTC on Bitcoin, Alice wants Bob's BTC and Bob wants Alice's ETH, the execution of a swap would be like this:
+
+1. Alice's sets up by generating a secret and hashing it. She sends to Bob:
+    1. Hash of the secret. 
+    1. Her refund address on Ethereum.
+    1. Her redeem address on Bitcoin.
+    1. Proposed expiry times on the for the [HTLC](#htlc)s. (Note: That could also be done by Bob.)
+1. Bob receives the swap proposed by Alice and sends back (if he accepts):
```

Comment:
🤔 I find it confusing to point out that Bob has to accept given that this is the _execution_ phase and they have previously negotiated the swap parameters. Either party could obviously not continue with the execution protocol at any point, but I think what you are describing here is the successful execution of a HTLC-base atomic swap. 

Maybe explicitly state that on line 20 and then ignore all the unhappy paths.